component/attr_cache/attr_cache.go

```diff
@@ -521,7 +521,7 @@ func NewAttrCacheComponent() internal.Component {
 // On init register this component to pipeline and supply your constructor
 func init() {
 	internal.AddComponent(compName, NewAttrCacheComponent)
-	attrCacheTimeout := config.AddInt32Flag("attr-cache-timeout", int32(defaultAttrCacheTimeout), "attribute cache timeout")
+	attrCacheTimeout := config.AddUint32Flag("attr-cache-timeout", 0, "attribute cache timeout")
```

Comment:
Now we can restore this 0 to its original default value right?

Reply:
done